__tests__/shared/keyManager.ts

```diff
@@ -1,7 +1,10 @@
 import { IAgentOptions, IDIDManager, IKeyManager, TAgent, TKeyType } from '../../packages/core/src'
 import { computeAddress, serialize } from '@ethersproject/transactions'
+import { mapIdentifierKeysToDoc } from '@veramo/utils/src'
+import { IResolver } from '@veramo/core'
```

Comment:
Test imports should use the path import to avoid mismatched versions of veramo

Reply:
good catch